src/main/java/org/dependencytrack/persistence/DefaultObjectGenerator.java

```diff
@@ -289,4 +284,20 @@ private void loadDefaultNotificationPublishers() {
             }
         }
     }
+
+    /**
+     * Loads the default OSV ecosystems
+     */
+    private void loadDefaultOsvEcosystems() {
+        try (QueryManager qm = new QueryManager()) {
+            LOGGER.info("Synchronizing OSV ecosystems to datastore");
+            List<String> ecosystems = OsvDownloadTask.getEcosystems();
+            if(!ecosystems.isEmpty()) {
+                ecosystems.forEach(ecosystem -> {
+                    LOGGER.debug("Creating config property: " + OsvDownloadTask.OSV_CONFIG_GROUP + " / " + ecosystem);
+                    qm.createConfigProperty(OsvDownloadTask.OSV_CONFIG_GROUP, ecosystem, "false", IConfigProperty.PropertyType.BOOLEAN, ecosystem);
+                });
+            }
+        }
+    }
```

Comment:
Do we really need this? Without having tested it, the code looks like it should be able to work just fine without the ecosystem properties being pre-populated.

Reply:
Right, we can remove the default generator here. We'll need a new endpoint in ConfigPropertyResource to **upsert** the config property so that frontend can call that endpoint for osv ecosystem selection. 
Currently, it just updates existing config properties. Anyway we'll need to create a config property (from frontend) whenever user selects a newly added ecosystem from OSV.